pkg/logging/logging.go

```diff
@@ -57,24 +57,27 @@ type LogOptions struct {
 // SetLogOptions set the LoggingOptions of NetConf
 func SetLogOptions(options *LogOptions) {
 	// give some default value
-	logger.MaxSize = 100
-	logger.MaxAge = 5
-	logger.MaxBackups = 5
-	logger.Compress = true
+	var updatedLogger lumberjack.Logger = lumberjack.Logger{Filename: logger.Filename, MaxAge: logger.MaxAge, MaxBackups: logger.MaxBackups,
+		Compress: logger.Compress, MaxSize: logger.MaxSize, LocalTime: logger.LocalTime}
```

Comment:
sorry for not raising it before, but do you really need to initialize these ? You're over-writing the defaults just below.

I mean, couldn't you just:
```golang
var updatedLogger lumberjack.Logger = lumberjack.Logger{
    Filename: logger.Filename, 
    MaxAge: 5,
    MaxBackups: 5,
    Compress: true,
    MaxSize: 100,
    LocalTime: logger.LocalTime
}
if options != nil {
...
}
...
```